src/sentry/digests/backends/redis.py

```diff
@@ -357,37 +371,70 @@ def try_lock(item):
 
                     pipeline.zrem(
                         make_schedule_key(self.namespace, SCHEDULE_STATE_READY),
-                        *[key for (lock, (key, timestamp)) in can_reschedule[True]]
+                        *[entry.key for (lock, entry) in can_reschedule[True]]
                     )
 
-                    pipeline.zadd(
-                        make_schedule_key(self.namespace, SCHEDULE_STATE_WAITING),
-                        *itertools.chain.from_iterable([(timestamp, key) for (lock, (key, timestamp)) in can_reschedule[True]])
+                    should_reschedule = {
```

Comment:
I guess the difference is negligible, if any. I thought maps were more expensive for lookups then an integer key for a list.